src/app/story/directives/basic.component.ts

```diff
@@ -24,7 +24,8 @@ import { StoryModel, TabService } from '../../shared';
         <label for="description">Description</label>
       </h3>
       <p class="hint">A full description of your piece.</p>
-      <publish-wysiwyg *ngIf="story" [model]="story" name="description" [images]="story.images"></publish-wysiwyg>
+      <publish-wysiwyg *ngIf="story" [model]="story" name="description" [content]="story.description" [images]="story.images" 
+        [changed]="descriptionChanged"></publish-wysiwyg>
```

Comment:
Could also have used just `story.changed('description')` here (since there's already an ng-if-story) ... but this is equivalent.